### ISSUE TITLE
Add event `NpcCrewWagePaid`

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#777]
## EDDI version in which issue found
3.0.1

## Investigation
Request from KellyR on the forums:
```
Is there any way we can get an event that will fire when an NPC crew member is paid and how much they are paid?

The information is included in the journal. Below is a sample journal entry.

{ "timestamp":"2018-08-07T06:34:29Z", "event":"NpcCrewPaidWage", "NpcCrewName":"Paloma Galloway", "NpcCrewId":140872258, "Amount":5301 }

Thanks for your consideration.
```

**Comment by Tkael** [comment]
Duplicate of #370

**Comment by Hoodathunk** [comment]
Included in PR #1282
